tests/dedicatedevhtests/l7_evh_crd_test.go

```diff
@@ -0,0 +1,440 @@
+/*
```

Comment:
Makefile changes seem to be missing for this folder.

Reply:
Added the required changes.